src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -240,7 +249,9 @@ export default {
 		},
 
 		isLinkComponent() {
-			return this.hasUrl ? 'a' : 'div'
+			return this.hasUrl
+				? (this.to ? RouterLink : 'a')
+				: 'div'
```

Comment:
Isn't this logic incorrect? If it's supposed to be a `RouterLink` it only has `to`, but not `url` set. But here both would be required to become a `RouterLink`.
I think this would be the correct solution, if we want to give precedence to `url` over `to`:
```suggestion
			return this.hasUrl
				? 'a'
				: (this.to ? RouterLink : 'div')
```
Or did I get something wrong here?

Reply:
No, seems I mislooked it